container_images.mk

```diff
@@ -49,3 +49,6 @@ SALT_MASTER_BUILD = 1
 SALT_MASTER_IMAGE_NAME = salt-master
 SALT_MASTER_IMAGE_SALT_VERSION = 2018.3.3
 SALT_MASTER_IMAGE_VERSION = $(SALT_MASTER_IMAGE_SALT_VERSION)-$(SALT_MASTER_BUILD)
+
+PLATFORM_UI_IMAGE_NAME = platform-ui
+PLATFORM_UI_IMAGE_VERSION = 0.2
```

Comment:
Where does this number come from?

Reply:
It's a random number. We can change it.
What version number do you want ?

Reply:
Take `0.1` I guess

Reply:
The version will be fix in a debt ticket. #819